examples/webgl_postprocessing_afterimage.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - postprocessing - afterimage</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<style>
+			body {
+				margin: 0px;
+				background-color: #000000;
+				overflow: hidden;
+			}
+		</style>
+	</head>
+	<body>
+
+		<script src="../build/three.js"></script>
+
+		<script src="js/shaders/CopyShader.js"></script>
+		<script src="js/shaders/AfterimageShader.js"></script>
+
+		<script src="js/postprocessing/EffectComposer.js"></script>
+		<script src="js/postprocessing/RenderPass.js"></script>
+		<script src="js/postprocessing/MaskPass.js"></script>
+		<script src="js/postprocessing/ShaderPass.js"></script>
+		<script src="js/postprocessing/AfterimagePass.js"></script>
+		
+		<script src="js/libs/dat.gui.min.js" type="text/javascript" charset="utf-8"></script>
+
+		<script>
+
+			var camera, scene, renderer, composer;
+			var mesh, light;
+
+			var afterimagePass, turnOnPass = true;
```

Comment:
I would call it just `enable` so it's consistent to: https://threejs.org/examples/#webgl_postprocessing_sobel

Please also update the label of the `dat.gui` interface.